.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 5.9.3
     hooks:
       - id: isort
-        args: [--filter-files,  --check-only]
+        args: [--filter-files, --check-only]
         files: \.py$
```

Comment:
We can limit the files like this to inclde only .md, .yaml, .yml, .json and .ipynb files. I think its a good idea to just limit it to these because we are using black for all other things.